components/blocks/SearchBar/SearchBar.test.ts

```diff
@@ -21,6 +29,8 @@ describe('<SearchBar />', () => {
       await fireEvent.click(getByTestId('search-button'))
 
       expect(emitted().search).toBeTruthy()
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error
       expect(emitted().search?.[0]?.[0]).toEqual(inputValue)
```

Comment:
What's the TS error that we get here?

Reply:
```Element implicitly has an 'any' type because expression of type '0' can't be used to index type '{}'.
  Property '0' does not exist on type '{}'.ts(7053)```

Reply:
Do you think it'll be more effort than it's worth/even possible to type this out? If either one's a no, then yeah let's just resolve this thread.

Reply:
Right now I don't see a need, especially with the tests not working. But whenever we get tests working maybe I can get better type inference in these tests such that we can have proper typing.